homeassistant/components/edl21/sensor.py

```diff
@@ -56,8 +87,17 @@ def event(self, message_body) -> None:
 
             device = self._cache.get(obis)
             if not device:
-                device = self._cache[obis] = EDL21Entity(obis, telegram)
-                new_devices.append(device)
+                name = self._OBIS_NAMES.get(obis)
+                if name:
+                    device = self._cache[obis] = EDL21Entity(obis, name, telegram)
+                    new_devices.append(device)
+                elif obis not in self._blacklist:
+                    _LOGGER.warning(
+                        "Unhandled sensor %s detected. Please report at "
+                        "https://github.com/home-assistant/home-assistant/issues",
```

Comment:
Let's filter the issues by this integration.

```suggestion
                        "https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+label%3A"integration%3A+edl21"+",
```